test/conformance/virtual_memory/urPhysicalMemGetInfo.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2023 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <uur/fixtures.h>
+
+using urPhysicalMemGetInfoWithFlagsParamTest =
+    uur::urPhysicalMemTestWithParam<ur_physical_mem_info_t>;
+UUR_TEST_SUITE_P(urPhysicalMemGetInfoWithFlagsParamTest,
+                 ::testing::Values(UR_PHYSICAL_MEM_INFO_REFERENCE_COUNT),
+                 uur::deviceTestWithParamPrinter<ur_physical_mem_info_t>);
+
+TEST_P(urPhysicalMemGetInfoWithFlagsParamTest, Success) {
+    size_t info_size = 0;
+    ur_physical_mem_info_t info = getParam();
+    ASSERT_SUCCESS(
+        urPhysicalMemGetInfo(physical_mem, info, 0, nullptr, &info_size));
+    ASSERT_NE(info_size, 0);
+
+    std::vector<uint8_t> data(info_size);
+    ASSERT_SUCCESS(urPhysicalMemGetInfo(physical_mem, info, data.size(),
+                                        data.data(), nullptr));
+
+    switch (info) {
```

Comment:
So GetInfo has been done with this structure in most other components so I was just following that practice. We could create a new issue to take a pass change that later? I'm guessing it was done this way because if you take a look at DeviceGetInfo there are around 120 properties to check

Reply:
I think we should start with making the small change for these new tests and have a ticket for making the wider change.

Reply:
Sure, will update this and make the ticket for updating the other tests.

Reply:
I've split this out now, and opened an issue to come back to later for all the other components - https://github.com/oneapi-src/unified-runtime/issues/2290